definitions/video-datazoom_video/definition.yml

```diff
@@ -0,0 +1,29 @@
+domain: VIDEO
+# Represents the video player experience across all instances of a customer's Browser Video players instrumented with New Relic's Video SDKs
+type: DATAZOOM_VIDEO
+
+synthesis:
+  rules:
+    - identifier: configuration_name
+      name: configuration_name
+
+      encodeIdentifierInGUID: false
```

Comment:
can we set this to true?

I believe most or all of the times the name will be short but I also don't think there's anything preventing it to go over the limit right?